### PR TITLE
State of the Art finished. Conclusions Improved.

### PR DESCRIPTION
**Pull request by amorag**
I have improved some sections and revised the whole paper.

There are several suggestions and comments to address with the label
"Antonio".

### CODE REVIEW

**Comment by JJ** [comment]
Please *always* pull before you do the pull request. Now your copy is in conflict, you'll have to solve it.

**Comment by amorag** [comment]
I have solved the conflicts.
Please accept my pull request now.

File: evostar2019/angrybirds-evo19.tex [hunk 1]
```diff
@@ -1185,70 +1144,71 @@ \section{Conclusions and Future Work}
...
+competitive entry for the Angry Birds level generation contest, and a
+secondary objective in mind: exploring the expressiveness and variability of SBPCG with evolutionary techniques. 
+%Perhaps the level of achievement of this one
+%is not as easily measurable as the other three. It could seem like
+%this objective has not been fully accomplished: only one SBPCG method
+%has been implemented and tested. However, it was not in the scope of
+%the project to test SBPCG in general, but in this particular case, for
+%this particular game. Considering this, 
+% Antonio - Do not say what you haven't done
+For this aim we have implemented and Evolutionary Algorithm able to optimize level structures to meet some criteria, being the main one the stability of the constructions. The method studied let us conclude that SBPCG methods are a potential good solution to offline content generation but it requires a great amount of problem-specific knowledge. Like any other form of creative work, the biggest issue may be how to measure how good, creative or enjoyable is the piece. The more rules the author adds, expressiveness starts to get lost as the
```
Comment (JJ) [inline evostar2019/angrybirds-evo19.tex]: It's a discussion section, we can say what we have achieve and what we haven't

File: evostar2019/angrybirds-evo19.tex [hunk 2]
```diff
@@ -1185,70 +1144,71 @@ \section{Conclusions and Future Work}
...
+%structured. Although the original intention was to make it available on Linux, 
+%currently it can only be executed with the desired behaviour on Windows. 
+%Another issue is that the simulation is not easily adaptable. If the fitness 
+%function of the EA is changed and needs data that is not included in the output 
+%right now, the game would have to be changed and compiled again. It
+%would be interesting to obtain other kind of data from the simulator,
+%such as the height of the structure and, eventually, its resistance to
+%bombardment by angry birds. However, this is left as future work.
+
+% Antonio - Again, in a paper conclusions you must remark what you have done. ;)
```
Comment (JJ) [inline evostar2019/angrybirds-evo19.tex]: It's conclusion and discussion. We can say what we have achieved, and why, and what we have not, and why.

File: evostar2019/angrybirds-evo19.tex [hunk 3]
```diff
@@ -1286,13 +1242,13 @@ \section{Conclusions and Future Work}
...
+Another important issue that will be addressed is the time performance. 
 Right now the simulation is the main bottleneck in the execution. One way to 
 speed up the process can be \textit{cleaning} up the current simulation, getting 
 rid of any unnecessary assets while maintaining the bare minimum to
-perform evaluation. However, this 
-will not solve the problem, since the content generator will still need to 
-launch an external executable.
+perform evaluation. 
+%However, this will not solve the problem, since the content generator will still need to launch an external executable.
+% Antonio - Do not mention future problems. ;D
```
Comment (JJ) [inline evostar2019/angrybirds-evo19.tex]: Why not? You have to show you have a grasp of the issues.

**Comment by JJ** [comment]
I'm closing this pull request. Please re-use whatever is reusable in the next iteration of the paper.

**Pull request closed without merging**